doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -362,22 +370,26 @@ In the case of being present, the error payload is JSON object including the fol
 + `affectedItems` (optional, array[string]): a list of elements affected by the error. Depending on the operation, it may
   refer to entities, registrations or subscriptions.
 
+All NGSIv2 server implementations must use the following HTTP response codes and `error` texts. However, the particular text used for
+`description` field is an implementation specific aspect.
+
 Error list (HTTP response code in parenthesis):
 
 * ParseError            (`400`). The incoming JSON payload cannot be parsed.
 * BadRequest            (`400`). The incoming request is invalid in this context.
-* NotFound              (`404`). The Context Element referred in the request has not been found.
+* NotFound              (`404`). The resource (entity, subscription, etc.) referred in the request has not been found.
 * TooManyResults        (`409`). There are several results that match with the resource identification used
-  in the request. This typically the case of requesting an entity with not enough information and the
-  solution is to enhance entity identification adding more information, e.g. adding entity type
-  and/or service path.
+  in the request. This is typically the case of requesting an entity with not enough information and the
```

Comment:
Fixed in 77dd58c.

Actually, I have removed the example based in entities, to make the description more general.